app/jobs/regular/process_bulk_invite_emails.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_dependency 'email/sender'
+
+module Jobs
+
+  class ProcessBulkInviteEmails < Jobs::Base
+
+    def execute(args)
+      bulk_invite_email_key = Invite::BULK_INVITE_EMAIL_KEY
+      pending_invite_ids = Invite.where(emailed_status: Invite.emailed_status_types[:pending]).limit(Invite::BULK_INVITE_EMAIL_LIMIT).pluck(:id)
+
+      if pending_invite_ids.length > 0
+        Invite.where(id: pending_invite_ids).update_all(emailed_status: Invite.emailed_status_types[:sending])
+        pending_invite_ids.each do |invite_id|
+          Jobs.enqueue(:invite_email, invite_id: invite_id)
+        end
+        Invite.where(id: pending_invite_ids).update_all(via_email: true, emailed_status: Invite.emailed_status_types[:sent])
```

Comment:
`Jobs.enqueue` should be sending the emails asynchronously so you can't update immediately afterwards and know that the emails were all sent. Instead the status should be updated by the `:invite_email` job.

Reply:
Understood, will update PR to do so.

Reply:
PR updated.